### PR TITLE
Create rhythmfpv.md

### PR DESCRIPTION
**Pull request by rhythm-fpv**
name: "rhythm-fpv"
project: "Racing FPV Drone Build"

# describe your project in 1-3 sentences
its a superfast fpv racing drone that defines the upcoming future of this hobby. FPV is a very interesting and
sophisticated hobby which is growing at an unimaginable rate, read my application to know more

its not a pre assembled kit, it requires a lot of experience with soldering and small electronic components to make a fpv quad 
which I have, i learn something new from fpv quads everyday, a build usually takes around 2-3 days to complete depending on size and availability, also an FPV quad can be modified in so many ways a person cant think of!
# Checklist before submitting proposal:

- [x]  I am a current high school student (or younger).
- [x] I have submitted an RSVP at https://hackclub.com/winter
- [x] I am a member of the Hack Club Slack (Join at https://hackclub.com/slack)
- [x] I have self-reviewed my project proposal.

[rhythmfpv.md](https://github.com/hackclub/winter/files/10421135/rhythmfpv.md)

### CODE REVIEW

**Comment by bellesea** [comment]
heyy thanks so much for making a PR! **we're back on tuesday and will start reviewing PRs in the order that they came in, thanks for your patience :)**

to make the process smoother, please ensure you double checked that your file is properly formatted, you have all the parts you need, and that your plan is detailed enough for us to imagine how you're using the components in your budget to build your project and visualise you work on it :)

and fyi, we're looking for projects where/that:
- you can spend at least 10 days building (from Feb 14th to 23rd)
- you are able to modify or make your own in some ways
- you can learn something new from
- involves working with electronics or software on a **lower level than a pre-assembled kit of parts & ready software**

and, we normally don’t approve projects where you buy consumer components w/o modification like keyboards (buying PCB, keycaps, switches, and assembling it), PCs (buying motherboard, cpu, memory and plugging it together), and kits (eg. closed-sourced consumer drone kits). 

_(note: can you please format your file similar to the sample in the [README](https://github.com/hackclub/winter#readme)? also, can you please factor in shipping & taxes in your budget? and, are you referencing any guides of sorts?)_

our Jan 15 deadline is for PRs to come in and not for final approval so we'll work with you to either accept this project OR give you a chance to submit another PR. 

if you have any questions, please do not hesitate to ask her or DM me (i'm belle on slack), thank you!

**Review by bellesea — CHANGES_REQUESTED** [review]
hey @rhythm-fpv can you please format your file similar to the sample in the [README](https://github.com/hackclub/winter#readme)? also, can you please add the links to your budget and factor in shipping & taxes in your budget? and, are you referencing any guides of sorts?

i also noticed you don't have wires, screws, etc. do you already have access to that?

thanks!

**Comment by rhythm-fpv** [comment]
> hey @rhythm-fpv can you please format your file similar to the sample in the [README](https://github.com/hackclub/winter#readme)? also, can you please add the links to your budget and factor in shipping & taxes in your budget? and, are you referencing any guides of sorts?
> 
> i also noticed you don't have wires, screws, etc. do you already have access to that?
> 
> thanks!

hey, thanks for your response. I'll do the mentioned steps for sure. second of all, the prices listed in the budget are incl. of shipping and taxes, also being a part of this hobby for a long time and being a student of science i have all kinds of wires and tools required to accomplish this project, also I have the basic knowledge of how to build my project but just in case if im stuck anywhere, the manuals come handy with all the different parts, also i must state a point that these are not just regular slow cinematic quads but they are capable to do literally anything and I bet you will find no one making such project here!!! thanks

File: rhythmfpv.md
Comment (bellesea) [inline rhythmfpv.md:8]:
```suggestion
I am building a racing fpv drone. So basically fpv is a whole new hobby in itself. Everyone builds thier own quadcopters does freestyle, The FPV community has recently started to grow in various parts of country.Companies like DJI, caddx, foxeer, TBS are a big contribution to this. New technologies like elrs are evolving and these superfast small compact drones are a huge oppurtunity for someone looking to enter a competetive and challenging hobby. The drone i am planning to make will fit well under 250usd.
```

File: rhythmfpv.md
Comment (bellesea) [inline rhythmfpv.md:25]: it looks like this link is also expired or smth as it redirects me to https://www.shorturl.at/

File: rhythmfpv.md
Comment (bellesea) [inline rhythmfpv.md:24]: this too haha...

**Review by bellesea — CHANGES_REQUESTED** [review]
@rhythm-fpv this is super cool and thanks for pushing the changes :) a few more questions hehe:
- can you please ensure the links are working? 100% cool if you want to put the full link instead of a shortlink
- how will you be viewing your video footage?
- will your drone be different from other drones in any other way? if yes which part did you change to fit what you want? 
- do you need propellers and batteries?

thanks!

**Comment by rhythm-fpv** [comment]
> @rhythm-fpv this is super cool and thanks for pushing the changes :) a few more questions hehe:
> 
> * can you please ensure the links are working? 100% cool if you want to put the full link instead of a shortlink
> * how will you be viewing your video footage?
> * will your drone be different from other drones in any other way? if yes which part did you change to fit what you want?
> * do you need propellers and batteries?
> 
> thanks!

Hey, Sorry for the inconvenience caused, the links were quite big but i've put the real ones now, hope it doesn't cause any problem, also to view the footage we will have a video transmitter on the quad which will then send signal to video reciever connected to our phone which will go into a virtual reality (VR)  box and then we can fly with the FPV experience, also if someone else also wants to see what we see then we have specialized FPV goggles just for this hobby(preowned).
Onto the third point, these drones are completely different, these are used in aggressive environments, we don't fly while seeing the drones, we fly by seeing, what the drone sees. Its a super fast speed quad, also can be used for videography purposes. Some professionals even tend to fly 20km far from the home point. Professional pilots like steele davis and nurk
have made it their full time profession. Its almost twice as small as normal drone giving it the speed. A professional high speed fpv quad can go upto 130mph. The model I have proposed to you can go upto 85-90mph.
Also being in this environment, batteries are most basic things to be owned and I have tons of em, that wouldn't cause any problem, same goes with props.

THANKS @bellesea LOOKING FORWARD FOR YOUR REPLY AND MAKING THE DEAL :-) :-)

File: rhythmfpv.md
Comment (bellesea) [inline rhythmfpv.md:1]:
```suggestion
---
name: "rhythm-fpv"
```

File: rhythmfpv.md
Comment (bellesea) [inline rhythmfpv.md:2]:
```suggestion
project: "Racing FPV Drone Build"
---
```

**Review by bellesea — CHANGES_REQUESTED** [review]
hiii @rhythm-fpv a few more questions:
- how would your video receiver connect to your phone?
- will you be writing any of your own code for this? thinking about whether you can modify this in anyway to make it more of your own hehe :)
- can you please add the parts you already have under budget and just put $0?

thank you sm!

**Comment by rhythm-fpv** [comment]
> hiii @rhythm-fpv a few more questions:
> 
> * how would your video receiver connect to your phone?
> * will you be writing any of your own code for this? thinking about whether you can modify this in anyway to make it more of your own hehe :)
> * can you please add the parts you already have under budget and just put $0?
> 
> thank you sm!

hi! receiver would connect through an USB cable and and app called GOFPV will then show us the video feed, 
Not so much of coding is required but yes in an open source app called betaflight, we need to program the flight controller very precisely and write a very few basic lines of codes to get some information. We also need to tune the quad that requires a lot of precision and experience with the quad. Also this quad is completely upto me, if I know about it ten I can do absolutely anything with it. Its completely manual and takes a lot of expertise to fly. Also one needs to have soldering skills to build these, so its completely my own quad, no need to worry on that, It will also help me to gain more experience.

I will add the parts right now.

So lets finalize the deal, what say? 😄😄

**Review by bellesea — APPROVED** [review]
@rhythm-fpv great! looking forward to you building this :)

a fabulous member of @hackclub/bank-ops will be reaching out soon with next steps!

approving to move forward but i hope that you can challenge yourself to expand beyond what you already know and perhaps modify this drone in the future OR write some of your own software for it. if you do so, please definitely update this file!

**Comment by rhythm-fpv** [comment]
> @rhythm-fpv great! looking forward to you building this :)
> 
> a fabulous member of @hackclub/bank-ops will be reaching out soon with next steps!
> 
> approving to move forward but i hope that you can challenge yourself to expand beyond what you already know and perhaps modify this drone in the future OR write some of your own software for it. if you do so, please definitely update this file!

hey thanks!! it was great talking to you, I will surely modify the drone as per my experience if it doesn't exceed the budget!

**Comment by rhythm-fpv** [comment]
> > @rhythm-fpv great! looking forward to you building this :)
> > a fabulous member of @hackclub/bank-ops will be reaching out soon with next steps!
> > approving to move forward but i hope that you can challenge yourself to expand beyond what you already know and perhaps modify this drone in the future OR write some of your own software for it. if you do so, please definitely update this file!
> 
> hey thanks!! it was great talking to you, I will surely modify the drone as per my experience if it doesn't exceed the budget!

I had a few questions, will I get the balance after the approval or there are more steps ahead? also will I be make payments if I live outside US (india)

**Review by daisybanks — APPROVED** [review]
Hi @rhythm-fpv! This sounds like a fun project and I'm excited to see you put your own twist on it! I'm dming you on Slack now to talk about next steps in getting you your grant! :)

**Review by daisybanks — COMMENTED** [review]
It actually looks like you're missing your Slack handle in your pr. Could you please dm me (@daisy) on Slack or comment your handle?

**Comment by rhythm-fpv** [comment]
> It actually looks like you're missing your Slack handle in your pr. Could you please dm me (@daisy) on Slack or comment your handle?

hey, sorry for being late, sorry but i dont know how to create a slack handle

**Comment by cfanoulis** [comment]
hey, are you a member of the [Hack Club Slack](https://hackclub.slack.com)?

**Comment by daisybanks** [comment]
I've messaged you on Slack at @fpvforlife. Please login to your Slack (hackclub.slack.com](http://hackclub.slack.com/) and reply so we can move forward with your project. :)

**Comment by cfanoulis** [comment]
hey @rhythm-fpv, congrats on the grant :) We've been trying to reach you for your ~car's extended warranty~ bank onboarding call - the next step to bring you close to your grant. However, your banker, Daisy, has been having difficulty reaching you to schedule a call. Can you please check [Slack](https://hackclub.slack.com/) & reply to their DM?

a bank onboarding call is crucial for the grant process, as this is where you'll be seeing Hack Club Bank for the first time (unless you have a demo account!). Also, you'll go through the Fiscal Sponsorship contract - a legal document officially getting you on da hack bank

Please make sure to schedule a meeting within the next 2 days (that is, until Jan 29th) - else, your grant will be forfeited! (and nobody wants that). If you're having trouble, please get in touch with either me or your banker via Slack.

Toodaloo!

**Pull request closed without merging**
